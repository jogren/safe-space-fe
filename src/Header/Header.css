.Header {
  height: 15vh;
  width: 100%;
  display: flex;
  flex-direction: column;
  border-bottom: 1px solid white;
}

#lotus-header {
  height: 50px;
  margin-top: 20px;
}

#text-header {
  color: white;
  margin: 0;
  text-align: center;
}