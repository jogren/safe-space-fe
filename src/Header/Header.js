import React from 'react'
import './Header.css'

export const Header = () => {
  return (
    <section className="Header">
      <a href ="/">
        <img alt="lotus-flower" id="lotus-header" src={require('../assets/lotus-flower.svg')}/>
        <h1 id="text-header">SafeSpace</h1>
      </a>
    </section>
  )
}

export default Header;
