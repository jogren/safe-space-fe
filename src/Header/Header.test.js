import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';


describe('Header', () => {
  let wrapper;

  beforeEach(() => {
    jest.resetModules()
    wrapper = shallow(<Header />)
  })

  it('should match the snapshot with the data passed through', () => {
    expect(wrapper).toMatchSnapshot();
  })
})