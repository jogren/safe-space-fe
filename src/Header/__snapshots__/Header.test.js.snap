// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot with the data passed through 1`] = `
<section
  className="Header"
>
  <img
    alt="lotus-flower"
    id="lotus-header"
    src={
      Object {
        "ReactComponent": Object {
          "$$typeof": Symbol(react.forward_ref),
          "render": [Function],
        },
        "__esModule": true,
        "default": "lotus-flower.svg",
      }
    }
  />
  <h1
    id="text-header"
  >
    SafeSpace
  </h1>
</section>
`;
